build/heroku-18.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM heroku/heroku:18-build.v98
```

Comment:
Do we want to keep the version here in the long term or is this for shor term only?

Reply:
Pinning to known versions of a stack image with a known list of available libraries is generally good practice to prevent a situation where libraries are added to newer stack image revisions that a future binary build then picks up automatically (because its `configure` auto-enables a shared library integration). Or just a library version update, should that happen in really exceptional cases.

For instance, in 2019, Ubuntu upgraded `libssl` on 18.04 from 1.1.0 to the LTS 1.1.1. Those two are compatible at runtime since they have the same ABI version, so something built against 1.1.0 works with 1.1.1, but if you build something against 1.1.1, it won't work with 1.1.0.

When we followed that upgrade in a stack image release, we had a new version with 1.1.1, but that was only on the Common Runtime - it would take Private Spaces a few days to follow, and during that time, Common Runtime had 1.1.1 which a Gem or a pre-built language runtime binary would compile against, while Private Spaces were still on 1.1.0, and so these binaries might call APIs that were specific to 1.1.1, since that's what they were compiled against, but executing that on Private Spaces with a runtime library still on 1.1.0 would lead to errors.

The solution was to pin to an older version of the stack image that still used 1.1.0 until Private Spaces had caught up, e.g.: https://github.com/heroku/heroku-buildpack-php/commit/a494884a5f7d391da47ed82f23c453e376f41ee6